akka-discovery/src/main/scala/akka/discovery/ServiceDiscovery.scala

```diff
@@ -297,10 +298,17 @@ abstract class ServiceDiscovery {
   /**
    * Scala API: Perform lookup using underlying discovery implementation.
    *
-   * Convenience for when only a name is required.
+   * Convenience lookup accepting a name. If the name is a valid srv entry a SRV lookup is done, otherwise
+   * if not a regular lookup. For more control use the overload accepting a [[Lookup]]
```

Comment:
```suggestion
   * Convenience lookup accepting a name. If the name is a valid SRV entry, an SRV lookup is done, otherwise
   * a regular lookup. For more control use the overload accepting a [[Lookup]].
```